apps/oxlint/src/command/lint.rs

```diff
@@ -62,14 +66,49 @@ pub struct BasicOptions {
     /// Oxlint configuration file (experimental)
     ///  * only `.json` extension is supported
     ///  * tries to be compatible with the ESLint v8's format
+    ///
+    /// If not provided, Oxlint will look for the following files in the current
+    /// directory (in order):
+    ///
+    /// - `.oxlintrc.json`
+    /// - `oxlintrc.json`
```

Comment:
I can't understand how we can guarantee the order here, What am I missing?
If we don't search with the order in mind we can drop the `(in order)` part.